java/hello-world/README.md

```diff
@@ -0,0 +1,43 @@
+# Cloud Bigtable Hello World
+
+This is a simple application that demonstrates using the native HBase API
+to connect to and interact with Cloud Bigtable.
+
+## Provision a cluster
+
+Follow the instructions in the [user documentation](https://cloud.google.com/bigtable/docs/creating-cluster)
+to create a Google Cloud Platform project and Cloud Bigtable cluster if necessary.
+You'll need to reference your project id, zone and cluster id to run the application.
+
+## Run the application
+
+First, set your [Google Application Default Credentials](https://developers.google.com/identity/protocols/application-default-credentials)
+and [install Maven](http://maven.apache.org/guides/getting-started/maven-in-five-minutes.html) if necessary.
+
+
+    $ mvn package
+    $ mvn exec:java -Dexec.args="<projectId> <zone> <clusterId>"
+
+You will see output resembling the following, interspersed with informational logging
```

Comment:
@sduskis This is also relevant to your comment on the outdated PR, but I added the <activation/> tag for each profile that automatically chooses the correct profile based on on the os family. Seems to work well on my mac:

garyelliott-macbookpro:hello-world garyelliott$ mvn help:active-profiles
[INFO] 
Active Profiles for Project 'com.example.bigtable.helloworld:cloud-bigtable-hello-world:jar:1.0-SNAPSHOT': 

The following profiles are active:

 - mac (source: com.example.bigtable.helloworld:cloud-bigtable-hello-world:1.0-SNAPSHOT)